src/SourceEngine.Demo.Parser/Events.cs

```diff
@@ -552,6 +555,9 @@ public static EquipmentElement MapEquipment(string name)
 			if (name.Contains("knife"))
 				return EquipmentElement.Knife;
 
+			if (name == "axe" || name == "hammer" || name == "spanner")
+				return EquipmentElement.Melee;
```

Comment:
This could be moved into the `switch` using multiple cases above a block (for example, `decoy` does this).

Reply:
not got a clue why I did this since I added everything else to the switch, probably a late night change!